lib/buffer.js

```diff
@@ -832,19 +839,17 @@ Buffer.prototype.fill = function fill(val, start, end, encoding) {
   }
 
   // Invalid ranges are not set to a default, so can range check early.
```

Comment:
Should it say "so **one** can range check early."?

Reply:
Or maybe 

`// Early range check because invalid ranges are not set to a default.`